--- origsrc/exo-0.3.2/exo/exo-mount-point.c	2007-01-20 07:58:22.000000000 -0600
+++ src/exo-0.3.2/exo/exo-mount-point.c	2007-05-10 23:17:53.031250000 -0500
@@ -193,7 +193,9 @@
 {
   GSList *mount_points = NULL;
 
-#if defined(HAVE_SETMNTENT) /* Linux */
+#if defined(__CYGWIN__)
+  return NULL;
+#elif defined(HAVE_SETMNTENT) /* Linux */
   struct mntent *mntent;
   FILE          *fp;
 
@@ -295,7 +297,9 @@
 {
   GSList *mount_points = NULL;
 
-#if defined(HAVE_SETMNTENT) /* Linux */
+#if defined(__CYGWIN__)
+  return NULL;
+#elif defined(HAVE_SETMNTENT) /* Linux */
   struct mntent *mntent;
   FILE          *fp;
 
--- origsrc/exo-0.3.2/python/Makefile.am	2007-01-20 07:58:18.000000000 -0600
+++ src/exo-0.3.2/python/Makefile.am	2007-05-06 21:48:21.437500000 -0500
@@ -31,7 +31,8 @@
 _exo_la_LIBADD =							\
 	$(top_builddir)/exo/libexo-$(LIBEXO_VERSION_API).la		\
 	$(GTK_LIBS)							\
-	$(LIBXFCE4UTIL_LIBS)
+	$(LIBXFCE4UTIL_LIBS)				\
+	$(PYTHON_LIBS)
 
 _exo_la_SOURCES =							\
 	exomodule.c							\
